### DISCUSSION TITLE
Proposals for a wofi replacement

### DISCUSSION THREAD

**Discussion by FilippoBonazziSUSE** [openSUSE/openSUSEway#140] [Ideas]
As we have discussed over several issues and PRs, there might be a need or a chance to replace `wofi` with something better.

[wofi](https://hg.sr.ht/~scoopta/wofi) appeared to be somewhat abandoned for some time, and is now in limited maintenance mode.

[rofi](https://github.com/davatorium/rofi) has been proposed as a concrete alternative.

Let's use this discussion to carry this topic forward.

**Comment by denisok** [comment]
I don't really care, I use only basic function - launch application and flatpak apps. Anything works for me if I can easily launch app

**Comment by FilippoBonazziSUSE** [comment]
Sway 1.9 has changed the default menu to [wmenu](https://git.sr.ht/~adnano/wmenu) (https://github.com/swaywm/sway/pull/7384)

**Comment by kskarthik** [comment]
Fuzzel is a good replacement for Wofi & wofi is not maintained anymore

**Comment by FilippoBonazziSUSE** [comment]
@staypirate recommended https://github.com/j0ru/kickoff
